src/ffi/types.rs

```diff
@@ -216,21 +216,21 @@ map_enum! {
 // 2 levels
 map_enum! {
     Level -> telio_log_level,
-    Error = TELIO_LOG_ERROR,
-    Warn = TELIO_LOG_WARNING,
-    Info = TELIO_LOG_INFO,
-    Debug = TELIO_LOG_DEBUG,
-    Trace = TELIO_LOG_TRACE,
+    ERROR = TELIO_LOG_ERROR,
+    WARN = TELIO_LOG_WARNING,
+    INFO = TELIO_LOG_INFO,
+    DEBUG = TELIO_LOG_DEBUG,
+    TRACE = TELIO_LOG_TRACE,
 }
 
 map_enum! {
     telio_log_level -> Level,
-    TELIO_LOG_CRITICAL = Error,
-    TELIO_LOG_ERROR = Error,
-    TELIO_LOG_WARNING = Warn,
-    TELIO_LOG_INFO = Info,
-    TELIO_LOG_DEBUG = Debug,
-    TELIO_LOG_TRACE = Trace,
+    TELIO_LOG_CRITICAL = ERROR,
```

Comment:
log_critical was used when we were using slog crate. Does it makes sense for us to remove this entirely now ?